jpf-symbc/src/main/gov/nasa/jpf/symbc/bytecode/IFNONNULL.java

```diff
@@ -3,16 +3,16 @@
  * Administrator of the National Aeronautics and Space Administration.
  * All rights reserved.
  *
- * Symbolic Pathfinder (jpf-symbc) is licensed under the Apache License, 
+ * Symbolic Pathfinder (jpf-symbc) is licensed under the Apache License,
```

Comment:
Please do not add formatting changes with your pull request, i.e., changes in spaces, etc should be ignored and not part of the pull request. We should only see changes in the code.